src/styles/breakpoints.js

```diff
@@ -11,7 +20,6 @@ export default function createBreakpoints(
   unit = 'px',
   step = 1,
 ) {
-  const keys = Object.keys(breakpoints);
```

Comment:
the spec says that `Object.keys` as the same order than `for...in`. That order is arbitrary